zeyple/zeyple.py

```diff
@@ -12,6 +12,12 @@
 import gpgme
 import copy
 from io import BytesIO
+import dns.resolver
```

Comment:
What is this import? it looks like a third party library. Is it in the standard library? I'm not very keen on adding new dependencies to the project.